sitemedia/css/admin-local.css

```diff
@@ -221,3 +221,8 @@ a#needsreview:target {
     clip: rect(0, 0, 0, 0);
     white-space: nowrap;
 }
+
+/* selected side thumbnail for fragment admin */
+img.selected {
+    border: 5px solid var(--accent);
```

Comment:
I had to look to see that this is defined in the django admin base styles, interesting. I wonder if it's visible enough / distinct enough from the colors in the images, but I don't see another obvious choice in the existing color variables so let's go ahead and try it.